src/components/molecules/Pagination.tsx

```diff
@@ -56,7 +56,7 @@ export default function Pagination({
 
   return totalPageNumbers && totalPageNumbers > 1 ? (
     <ReactPaginate
-      pageCount={totalPageNumbers}
+      pageCount={totalPageNumbers > 476 ? 476 : totalPageNumbers}
```

Comment:
it would be nice if the 476 value would be wrapped in a constant that we can use across all affected components, so if the maximum result value changes we don't need to find all the 476 values and replace them with the new total page number value and would be easier to understand in a eventual later update

Reply:
yep this can be done, I think we can add a constant inside aquarius file and import it in those places